src/targets/services/polyfillFetch.js

```diff
@@ -0,0 +1,16 @@
+import fetch from 'node-fetch'
+import FormData from 'form-data'
+import { URL } from 'url'
+
+const polyfillFetch = () => {
```

Comment:
Maybe you can add a comment to explain why/when this polyfill is needed

Reply:
See https://github.com/cozy/cozy-home/commit/77fa6b5a897d95fb16d89b162508756e49bfb018